src/controlCenter/widgets/backlight/backlightUtil.vala

```diff
@@ -77,7 +77,7 @@ namespace SwayNotificationCenter.Widgets {
             if (monitor != null) monitor.cancel ();
         }
 
-        public void set_brightness (float percent) {
+        public async void set_brightness (float percent) {
```

Comment:
You have to change the call to the methods to include a `.begin` (`implicit .begin is deprecated`)